### PR TITLE
tweaks for sprint demo to work locally

### PR DESCRIPTION
**Pull request by leahnp**
Since we are still working out the sprint cycle and demos, I want to add these notes and tweaks so I will remember how to demo my tickets.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -23,10 +23,10 @@ RUN go get github.com/golang/protobuf/protoc-gen-go
 
 RUN go get github.com/samsung-cnct/sample-grpc-apiserver/api
 
-RUN mkdir -p /usr/bin/api && \
-    cp -r . /usr/bin/api && \
-    cd /usr/bin/api && \
+RUN mkdir -p sample-grpc-apiserver/
+ADD . sample-grpc-apiserver/
+RUN cd sample-grpc-apiserver/ && \
     protoc -I api api/api.proto --go_out=plugins=grpc:api
```
Comment (alejandroEsc) [inline Dockerfile]: gateway code will change this command, fyi.

File: client/main.go [hunk 2]
```diff
@@ -29,7 +29,7 @@ import (
 )
 
 var (
-  serverAddr = flag.String("server_addr", "localhost:5300", "The server address in the format of host:port")
+  serverAddr = flag.String("server_addr", "0.0.0.0:5300", "The server address in the format of host:port")
```
Comment (alejandroEsc) [inline client/main.go]: please change this to 127.0.0.1

File: readme.md [hunk 3]
```diff
@@ -48,3 +48,8 @@ where what the above comes from is
 ```
 protoc --doc_out <output_folder> --doc_opt=<format, file_output> <api_proto_input_files>
 ```
+
+### Acceptance Criteria for BAC-130 & BAC-136
```
Comment (alejandroEsc) [inline readme.md]: please change title, do not add acceptance criteria, this is my opinion here but rather leave that at the ticket, not at the level of the readme file.

**Comment by leahnp** [comment]
No longer relevant for the demo

**Pull request closed without merging**